idea_town/experiments/migrations/0001_initial.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.conf import settings
+import idea_town.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Experiment',
+            fields=[
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', serialize=False, primary_key=True)),
+                ('title', models.CharField(max_length=128)),
+                ('slug', models.SlugField(unique=True, max_length=128)),
+                ('thumbnail', models.ImageField(upload_to=idea_town.utils.HashedUploadTo('thumbnail'))),
+                ('description', models.TextField()),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('modified', models.DateTimeField(auto_now=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='ExperimentDetail',
+            fields=[
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', serialize=False, primary_key=True)),
+                ('order', models.IntegerField(default=0)),
+                ('headline', models.CharField(max_length=256)),
+                ('image', models.ImageField(upload_to=idea_town.utils.HashedUploadTo('image'))),
+                ('copy', models.TextField()),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('modified', models.DateTimeField(auto_now=True)),
+                ('experiment', models.ForeignKey(related_name='details', to='experiments.Experiment')),
+            ],
+            options={
+                'ordering': ('experiment', 'order', 'modified'),
+            },
+        ),
+        migrations.CreateModel(
+            name='UserInstallation',
+            fields=[
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', serialize=False, primary_key=True)),
+                ('rating', models.FloatField(null=True)),
```

Comment:
And that was another of my questions I forgot to write down: What kind of input do we want to capture from users? 

I saw a 1-5 stars thing in wires, so I threw in rating. But there was also mention of a survey. We can expand the model for a quick comment field, but a survey is another app in and of itself (unless we, say, used an external service or something). This also plays into nailing down what metrics we might want to collect